package.json

```diff
@@ -1,159 +1,64 @@
-    "start": "next start",
-    "test": "vitest run",
+    "build": "lerna run build",
+    "clean": "lerna exec -- rm -rf node_modules build && rm -rf node_modules",
+    "commit": "git cz",
+    "download-zkeys:test": "lerna run download-zkeys:test --scope=maci-coordinator",
+    "download-zkeys:prod": "lerna run download-zkeys:prod --scope=maci-coordinator",
     "prettier": "prettier -c .",
     "prettier:fix": "prettier -w .",
-    "types": "tsc -p tsconfig.json --noEmit",
-    "eas:registerSchemas": "npx tsx src/lib/eas/registerSchemas",
-    "install:chromium": "playwright install chromium",
-    "test:e2e": "playwright test --project=chromium",
-    "prepare": "is-ci || husky"
-  },
-  "dependencies": {
-    "@ethereum-attestation-service/eas-sdk": "^1.5.0",
-    "@hookform/resolvers": "^3.3.4",
-    "@nivo/boxplot": "^0.84.0",
-    "@nivo/line": "^0.84.0",
-    "@pinata/sdk": "^2.1.0",
-    "@radix-ui/react-accordion": "^1.1.2",
-    "@radix-ui/react-dialog": "^1.0.5",
-    "@radix-ui/react-dropdown-menu": "^2.0.6",
-    "@rainbow-me/rainbowkit": "^2.0.1",
-    "@rainbow-me/rainbowkit-siwe-next-auth": "^0.4.0",
-    "@semaphore-protocol/core": "4.0.0-beta.16",
-    "@semaphore-protocol/data": "4.0.0-beta.16",
-    "@t3-oss/env-nextjs": "^0.8.0",
-    "@tailwindcss/forms": "^0.5.7",
-    "@tanstack/react-query": "^5.24.1",
-    "@testing-library/react": "^14.1.2",
-    "@trpc/client": "11.0.0-next-beta.294",
-    "@trpc/next": "11.0.0-next-beta.294",
-    "@trpc/react-query": "11.0.0-next-beta.294",
-    "@trpc/server": "11.0.0-next-beta.294",
-    "@vercel/blob": "^0.19.0",
-    "clsx": "^2.1.0",
-    "cmdk": "^0.2.0",
-    "date-fns": "^3.6.0",
-    "ethers": "^6.13.1",
-    "formidable": "^3.5.1",
-    "graphql-request": "^6.1.0",
-    "lowdb": "^1.0.0",
-    "lucide-react": "^0.316.0",
-    "maci-cli": "0.0.0-ci.4d2d340",
-    "maci-domainobjs": "0.0.0-ci.4d2d340",
-    "next": "^14.1.0",
-    "next-auth": "^4.24.5",
-    "next-themes": "^0.2.1",
-    "node-fetch-cache": "^4.1.0",
-    "nuqs": "^1.17.1",
-    "p-limit": "^5.0.0",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "react-hook-form": "^7.49.3",
-    "react-icons": "^5.0.1",
-    "react-markdown": "^9.0.1",
-    "react-number-format": "^5.3.1",
-    "react-use": "^17.5.0",
-    "siwe": "^2.1.4",
-    "sonner": "^1.4.0",
-    "superjson": "^2.2.1",
-    "tailwindcss": "^3.4.1",
-    "tailwind-merge": "^2.2.1",
-    "tailwind-variants": "^0.1.20",
-    "viem": "^2.7.15",
-    "wagmi": "^2.9.8",
-    "zod": "3.22.4"
+    "lint": "pnpm run lint:coordinator && pnpm run lint:interface",
+    "lint:coordinator": "eslint './packages/coordinator/**/*.ts'",
+    "lint:coordinator:fix": "eslint './packages/coordinator/**/*.ts' --fix",
+    "lint:interface": "lerna run lint --scope=maci-platform-interface",
+    "lint:interface:fix": "lerna run lint:fix --scope=maci-platform-interface",
+    "lint:fix": "pnpm run lint:coordinator:fix && pnpm run lint:interface:fix",
```

Comment:
It's better to do it like it's implemented for maci repo so we don't need to create command in root project to run linter/prettier and etc.